src/wwwroot/template-editor/adapterService.js

```diff
@@ -354,15 +356,28 @@
      */
     function getCampaign(id, useEditorAsTemplate) {
       traceAdapterCall(getCampaign, arguments);
-      var campaign = {
-        id: id,
-        type: 'campaign',
-        name: 'Campaign ' + id,
-        attributes: {},
-        innerHTML: '',
-        children: []
-      };
-      return $q.resolve({ data: campaign });
+      return $http.get(`${RELAY_CONFIG.baseUrl}/accounts/${loginSession.accountId}/template/${id}`, {
```

Comment:
It could be great using accountName in place of accountId. Probably you will need to use a URL template engine in place of simply concatenating strings, in order to take into account URL encoding and other weird cases.

Reply:
Do you have a favourite URL template engine  to use for this case? I was checking this:
https://github.com/bramstein/url-template

Reply:
In WebApp we are simply concatenating the sections, so, we do not need to do in better here. Keep the concatenation.